src/components/screens/OpeningTimesCard.js

```diff
@@ -1,7 +1,7 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import { View, StyleSheet } from 'react-native';
-import styled from 'styled-components';
+import styled from 'styled-components/native';
```

Comment:
what was the error here without `/native`? do you know why this fixes it? could this be necessary in other components as well?

Reply:
> what was the error here without `/native`? do you know why this fixes it? could this be necessary in other components as well?

The screenshot above shows the error, which tells that styled-components alone it's undefined , styled-components/native complete path is required, in order to use StyleSheet.create. It seem an anomaly present only in this file.

Reply:
did you select the wrong screenshot?
this error is not the one that I initially found and fixed.

Reply:
Yes you are right ! this is the error you have seen: 
![Screenshot 2021-03-19 at 18 18 08](https://user-images.githubusercontent.com/43641321/111818665-b7f05e80-88df-11eb-8b59-7e30c079a1f0.png)
but the screenshot above shows an error that I got as well from this issue